metrics/metricsreporter.go

```diff
@@ -135,27 +135,22 @@ func (m *MetricsReporter) CaptureRoutesPruned(routesPruned uint64) {
 	m.Batcher.BatchAddCounter("routes_pruned", routesPruned)
 }
 
-func (m *MetricsReporter) CaptureRegistryMessage(msg ComponentTagged) {
+func (m *MetricsReporter) CaptureRegistryMessage(msg ComponentTagged, action string) {
 	var componentName string
 	if msg.Component() == "" {
-		componentName = "registry_message"
+		componentName = "registry_message." + action
 	} else {
-		componentName = "registry_message." + msg.Component()
+		componentName = "registry_message." + action + "." + msg.Component()
```

Comment:
This will probably also change the metric right? I was hoping that we could preserve the metric name but only add an additional tag to it.

Reply:
[This](https://github.com/cloudfoundry/dropsonde/blob/cce4f49c37d11195c32193cc23fe09210d4ca18c/metricbatcher/metricbatcher.go#L195) is what I had in mind. But it seems like the API is a bit cumbersome to use...